Firestore/tests/Unit/Connection/GrpcTest.php

```diff
@@ -230,6 +230,29 @@ public function testListCollectionIds()
         $this->sendAndAssert('listCollectionIds', $args, $expected);
     }
 
+    public function testListCollectionIdsWithReadTime()
+    {
+        $args = [
+            'parent' => sprintf(
+                'projects/%s/databases/%s/documents',
+                self::PROJECT,
+                self::DATABASE
+            ),
+            'readTime' => [
+                'seconds' => (int) 123456789,
```

Comment:
we dont need to cast to int.. pls change here and at other places

Reply:
Changed in [41b2311](https://github.com/googleapis/google-cloud-php/pull/5865/commits/41b2311e14606643954616150399eaf1a13d0ba1). Thanks for pointing out.